Parsers/ASimAuditEvent/Parsers/ASimAuditEventBarracudaWAF.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
can we map below?
(1) Warning: Missing recommended field [EventUid]

Reply:
can we check this?
(1) Warning: Empty value in 150 records (100.0%)  in mandatory field [EventEndTime] (Schema:AuditEvent)
(1) Warning: Empty value in 150 records (100.0%)  in mandatory field [EventStartTime] (Schema:AuditEvent)

Reply:
done